kothic/style/mapcss.js

```diff
@@ -239,8 +239,86 @@ var MapCSS = {
 
 	e_cond: function (cond, a, b) {
 		return (cond) ? a : b;
+	}
+
+	e_split: function (sep, text) {
+		return text.split(sep);
+	},
+
+	e_get: function(arr, index) {
+		if (Object.prototype.toString.call(arr) !== '[object Array]')
+			return "";
+
+		if (!/^[0-9]+$/.test(index) || index >= arr.length())
+			return "";
+
+		return arr[index];
+	},
+
+	e_set: function(arr, index, text) {
+		if (Object.prototype.toString.call(arr) !== '[object Array]')
+			return [];
+
+		if (!/^[0-9]+$/.test(index))
+			return [];
+
+		arr[index] = text;
+
+		return arr;
+	},
+
+	e_count: function(arr) {
+		if (Object.prototype.toString.call(arr) !== '[object Array]')
+			return 0;
+
+		return arr.length();
+	},
+
+	e_list: function() {
+		return arguments;
 	},
 
+	e_append: function(lst, v) {
+		if (Object.prototype.toString.call(lst) !== '[object Array]')
+			return [];
+
+		lst.push(v);
+
+		return lst;
+	}
```

Comment:
A `,` is missing after the function (also for the following functions up to `e_reverse`).